ui/package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@material-ui/core": "^4.1.3",
```

Comment:
please avoid any ui library here, its not that hard to define own components for the needs of the feature

Reply:
the simplest example of the component is `components/Status`